interfaces/apparmor/apparmor.go

```diff
@@ -70,6 +75,13 @@ func loadProfiles(fnames []string, cacheDir string, flags aaParserFlags) error {
 
 	// Use no-expr-simplify since expr-simplify is actually slower on armhf (LP: #1383858)
 	args := []string{"--replace", "--write-cache", "-O", "no-expr-simplify", fmt.Sprintf("--cache-loc=%s", cacheDir)}
+	if flags&conserveCPU != 0 {
+		cpus := runtime.NumCPU()
+		if cpus > 1 {
+			// spare one CPU
```

Comment:
it might be a good idea to explain why we spare one CPU in a doc-comment I think

Reply:
why the choice of sparing one? we might usually be booting maybe at this point, there is a lot going on potentially? should we use half the cpus instead?

Reply:
I went ahead with @jdstrand suggestion of sparing two CPUs for the time being, I'm open to changing this. Also moved it to a helper.